content/en/docs/setup/upgrade/helm/index.md

```diff
@@ -37,6 +37,20 @@ To get started, check out https://istio.io/latest/docs/setup/getting-started/
 
 {{< boilerplate helm-backup >}}
 
+### Upgrading Custom Resource Definitions (CRDs)
+
+{{< warning >}}
+Helm does not [upgrade or delete](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations)
+CRDs when performing an upgrade. Given this restriction, additional measures are required to ensure that
+Istio is up to date when performing an upgrade.
+{{< /warning >}}
```

Comment:
```suggestion
[Helm does not upgrade or delete CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#some-caveats-and-explanations) when performing an upgrade. Because of this restriction, an additional step is required when upgrading Istio with Helm.
{{< /warning >}}
```